### ISSUE TITLE
module 'wishbone' has no attribute 'wb'

### ISSUE THREAD

**Issue by agelber** [ManuSetty/wishbone#37]
Hi,

I have just started looking into wishbone. However when I try to follow the tutorial I continually get the error: `module 'wishbone' has no attribute 'wb' `

Do you have any idea why this might be? I have anaconda installed with python 3.6.

Thanks,
Alon

**Comment by ManuSetty** [comment]
Hello

How did you install Wishbone -  Did you install from source?

**Comment by agelber** [comment]
Yes, I cloned it from your GitHub repository and then used pip3 to install it.  Not sure if this helps but I took a screenshot of the error and attached it here.

<img width="1114" alt="screen shot 2017-05-22 at 9 56 26 pm" src="https://cloud.githubusercontent.com/assets/24547894/26335421/bfde6932-3f39-11e7-8627-48deeeddae0a.png">

**Comment by ManuSetty** [comment]
I just tried a fresh install on my computer with Python 3.6. and unfortunately wasn't able to reproduce this issue. Can you give me more details about the platform on which you are running into this.

**Comment by agelber** [comment]
I'm not exactly sure why this was happening but I did finally get it working. I was able to recreate the issue on a mac and on a machine running ubuntu but I think it may have been because different dependencies were installed using different python versions. I'm not quite sure.
I do know that on a fresh python install, the wishbone pip3 install will fail on a Cython dependency issue, which the user then has to manually install. Maybe in the previous cases this was installed with Python 2 or something?
